src/components/Checkbox/Checkbox.jsx

```diff
@@ -22,29 +24,40 @@ class Checkbox extends React.Component {
     } = this.props;
     const id = uniqueId('id');
     const inputClassName = indeterminate ? style.indeterminate : style.original;
+    const wrapperClassName = cx({
+      [style.wrapper]: !disabled,
+      [style.wrapperDisabled]: disabled,
+    });
+    const textClassName = cx({
+      [style.textBold]: isBold,
+      [style.text]: !disabled,
+      [style.textDisabled]: disabled,
+    });
```

Comment:
Why don't you just do something like

```
cx(style.text, {
  [style.textBold]: isBold,
  [style.textDisabled]: disabled,
})
```

where `.text` would just be:

```
margin-left: 10px;
@extend %type--body;		   
cursor: pointer;
```

and you'd get rid if `.textBase` cause that's unnecessary

and then `.textDisabled` would become

```
color: $cGray200;
```

and if need be, add the disabled cursor to that last one?

Reply:
sounds good

Reply:
not sure what my motivation was, separating them into separate classes that each had the same thing, maybe it was a BEM thing 🤷‍♂️

Reply:
oh! now I remember, I didn't want to have the hover styling for the checkbox applying when the wrapper was set to disabled, and I didn't want to have to override that style with the default style. But I can definitely remove the repetition on the text classes